### PR TITLE
Chore/update network restrictions

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: studio/components/interfaces/Settings/Database/BannedIPs.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ const BannedIPs = () => {
     <div id="banned-ips">
       <div className="flex items-center justify-between">
         <FormHeader
-          title="Banned IPs"
+          title="Network Bans"
           description="We monitor unsuccessful logins and block IPs for security"
```
Comment (MildTomato) [inline studio/components/interfaces/Settings/Database/BannedIPs.tsx:68]:
```suggestion
          List of IP addresses that are temporarily blocked if their traffic pattern looks abusive.
```

File: studio/components/interfaces/Settings/Database/NetworkRestrictions/AddRestrictionModal.tsx [hunk 2]
```diff
@@ -34,7 +34,9 @@ const AddRestrictionModal = ({
 
     // Validate CIDR block size
     const isOutOfCidrSizeRange = cidrBlockSize < 0 || cidrBlockSize > 32
-    if (isOutOfCidrSizeRange) errors.cidrBlockSize = 'Size has to be between 0 to 32'
+    if (cidrBlockSize.length === 0 || isOutOfCidrSizeRange) {
```
Comment (alaister) [inline studio/components/interfaces/Settings/Database/NetworkRestrictions/AddRestrictionModal.tsx:37]: I think 0 may be valid value actually – maybe check with infra?

Comment (joshenlim) [reply studio/components/interfaces/Settings/Database/NetworkRestrictions/AddRestrictionModal.tsx:37]: yeap 0 is a valid value - the additional check here just verifies if the input field is empty

Comment (joshenlim) [reply studio/components/interfaces/Settings/Database/NetworkRestrictions/AddRestrictionModal.tsx:37]: so validation will say "no" if input field is empty, or a number below 0 or higher than 32 is given

Comment (alaister) [reply studio/components/interfaces/Settings/Database/NetworkRestrictions/AddRestrictionModal.tsx:37]: ah my bad! nice

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**